server/utils/datetime_utils.py

```
@@ -0,0 +1,162 @@
+    Return the current time (local or UTC) as ISO 8601 for DB storage.
+    Safe for SQLite, PostgreSQL, etc.
+
+    Example local: '2025-11-04 18:09:11'
+    Example UTC:   '2025-11-04 07:09:11'
+    """
+    if local:
+        try:
+            if isinstance(conf.tz, datetime.tzinfo):
+                tz = conf.tz
+            elif conf.tz:
+                tz = ZoneInfo(conf.tz)
```

Ruff (0.14.3)

`ZoneInfo` may be undefined, or defined from star imports

(F405)